Vagrantfile

```diff
@@ -35,9 +35,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision :shell, path: "deployment/bootstrap.sh"
 
   config.vm.provision :puppet do |puppet|
-    puppet.manifests_path = "deployment/puppet/manifests"
-    puppet.manifest_file  = "site.pp"
-    puppet.hiera_config_path = "deployment/puppet/hiera.yaml"  # to avoid missing file warning
+    puppet.environment_path = "deployment/puppet/environments"
```

Comment:
"Environments are isolated groups of Puppet agent nodes." https://puppet.com/docs/puppet/5.0/environments_about.html
A simple single manifest is sufficient for our purposes.